### PR TITLE
all: gofumpt

### PR DESCRIPTION
**Pull request by keegancsmith**
gofumpt is a stricter format than gofmt, while being backwards
compatible. That is, gofumpt is happy with a subset of the formats that gofmt
is happy with. See https://github.com/mvdan/gofumpt

```shell
gofumpt -w -s .
```

We won't enforce gofumpt output since that requires all developers of
sourcegraph to switch to it. Instead this commit is a once off (and may happen
again in the future).

### CODE REVIEW

**Comment by mrnugget** [comment]
Looks good! I mean, leaving aside the fact that the tool should be called `goformat`, of course.

File: cmd/frontend/internal/session/session_test.go [hunk 1]
```diff
@@ -208,17 +208,18 @@ func TestCookieMiddleware(t *testing.T) {
...
-		expActor: &actor.Actor{},
-	}, {
-		req:      authedReqs[0],
-		expActor: actors[0],
-	}, {
-		req:      authedReqs[1],
-		expActor: &actor.Actor{},
-		deleted:  true,
-	},
+	}{
```
Comment (tsenart) [inline cmd/frontend/internal/session/session_test.go]: @keegancsmith: Not sure you like this...  😸

File: cmd/repo-updater/repos/integration_test.go [hunk 2]
```diff
@@ -54,18 +54,30 @@ func TestIntegration(t *testing.T) {
...
-		{"Migrations/BitbucketServerUsername",
-			testBitbucketServerUsernameMigration(store)},
-		{"Migrations/AWSCodeCommitSetBogusGitCredentialsMigration",
-			testAWSCodeCommitSetBogusGitCredentialsMigration(store)},
-		{"Migrations/EnabledStateDeprecationMigration",
-			testEnabledStateDeprecationMigration(store)},
+		{
+			"Migrations/GithubSetDefaultRepositoryQuery",
+			testGithubSetDefaultRepositoryQueryMigration(store),
+		},
```
Comment (tsenart) [inline cmd/repo-updater/repos/integration_test.go]: OK, this is fine.

File: cmd/repo-updater/repos/store.go [hunk 3]
```diff
@@ -772,7 +772,6 @@ func scanRepo(r *Repo, s scanner) error {
 		&sources,
 		&metadata,
 	)
-
```
Comment (tsenart) [inline cmd/repo-updater/repos/store.go]: Not sure if this is a net improvement. The vertical space aids readability in these cases where the error returning function call is split in multiple lines.

File: cmd/frontend/internal/session/session_test.go (hunk 1 above)
Comment (keegancsmith) [reply cmd/frontend/internal/session/session_test.go]: looking at the file, looks like it was messed up anyways (see the red side just after the diff)

File: cmd/repo-updater/repos/store.go (hunk 3 above)
Comment (keegancsmith) [reply cmd/repo-updater/repos/store.go]: /me shrugs. I can remove it. I personally like the "rule" of no whitespace between err and check.

Comment (tsenart) [reply cmd/repo-updater/repos/store.go]: I can live with that.

**Pull request merged**